ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderElection.java

```diff
@@ -448,6 +570,7 @@ private ResultAndTerm waitForResults(Phase phase, long electionTerm, int submitt
         } else {
           rejectedPeers.add(replierId);
           if (conf.majorityRejectVotes(rejectedPeers)) {
+            LOG.info("rejectedPeers: {}", rejectedPeers);
```

Comment:
From this log, we can tell that a certain peer was rejected. But as a user, I would prefer to know why it was rejected. Can we change the log to something like the following instead?

```
LOG.info("rejectedPeers: {}, emptyCommit : {}, emptyLog : {}", 
    rejectedPeers, emptyCommit, nonEmptyLog(r));
```

Reply:
Sure, we can add `emptyCommit`. 

 For `nonEmptyLog(r)`, `r` is just one of responses but not all of them.  So, printing it will be confusing.